hack/bump-version.sh

```diff
@@ -1,31 +0,0 @@
-#!/bin/bash
```

Comment:
What is the purpose of this file? 
Did CFE or somebody else provide it?
Is is part of some existing CI?

Reply:
we copied and pasted everything from odo: https://github.com/openshift/odo

but cool support for the versioning, how do we do the release version? via git tag?

Reply:
Yes, it comes from a [tag](https://github.com/openshift/build-machinery-go/blob/2b271bb3a0ad466045cd6da5c9423084e9cf68f0/make/lib/golang.mk#L53): `git describe --long --tags --abbrev=7 --match 'v[0-9]*'`